src/main/kotlin/me/zeroeightsix/kami/feature/module/MurderMysteries.kt

```diff
@@ -0,0 +1,124 @@
+                        Items.GOLDEN_SWORD -> mc.player?.sendMessage(murdererIs(username, it.item.name.string), false)
+                        Items.WOODEN_SWORD -> mc.player?.sendMessage(murdererIs(username, it.item.name.string), false)
+
+                        Items.NETHERITE_AXE -> mc.player?.sendMessage(murdererIs(username, it.item.name.string), false)
+                        Items.DIAMOND_AXE -> mc.player?.sendMessage(murdererIs(username, it.item.name.string), false)
+                        Items.IRON_AXE -> mc.player?.sendMessage(murdererIs(username, it.item.name.string), false)
+                        Items.GOLDEN_AXE -> mc.player?.sendMessage(murdererIs(username, it.item.name.string), false)
+                        Items.WOODEN_AXE -> mc.player?.sendMessage(murdererIs(username, it.item.name.string), false)
+                    }
+
+                    //Check for tool items like shovels, hoes, and shears, and picks (hypixel knife skin)
+                    if (toolItems) when (it.item) {
+                        Items.NETHERITE_SHOVEL -> mc.player?.sendMessage(murdererIs(username, it.item.name.string), false)
+                        Items.DIAMOND_SHOVEL -> mc.player?.sendMessage(murdererIs(username, it.item.name.string), false)
+                        Items.IRON_SHOVEL -> mc.player?.sendMessage(murdererIs(username, it.item.name.string), false)
+                        Items.GOLDEN_SHOVEL -> mc.player?.sendMessage(murdererIs(username, it.item.name.string), false)
+                        Items.WOODEN_SHOVEL -> mc.player?.sendMessage(murdererIs(username, it.item.name.string), false)
+
+                        Items.NETHERITE_HOE -> mc.player?.sendMessage(murdererIs(username, it.item.name.string), false)
+                        Items.DIAMOND_HOE -> mc.player?.sendMessage(murdererIs(username, it.item.name.string), false)
+                        Items.IRON_HOE -> mc.player?.sendMessage(murdererIs(username, it.item.name.string), false)
+                        Items.GOLDEN_HOE -> mc.player?.sendMessage(murdererIs(username, it.item.name.string), false)
+                        Items.WOODEN_HOE -> mc.player?.sendMessage(murdererIs(username, it.item.name.string), false)
+
+                        Items.NETHERITE_PICKAXE -> mc.player?.sendMessage(murdererIs(username, it.item.name.string), false)
+                        Items.DIAMOND_PICKAXE -> mc.player?.sendMessage(murdererIs(username, it.item.name.string), false)
+                        Items.IRON_PICKAXE -> mc.player?.sendMessage(murdererIs(username, it.item.name.string), false)
+                        Items.GOLDEN_PICKAXE -> mc.player?.sendMessage(murdererIs(username, it.item.name.string), false)
+                        Items.WOODEN_PICKAXE -> mc.player?.sendMessage(murdererIs(username, it.item.name.string), false)
+
+                        Items.SHEARS -> mc.player?.sendMessage(murdererIs(username, it.item.name.string), false)
+                    }
```

Comment:
Shears are an exception, but all mining tools listed here are instances of `MiningToolItem`. Axes are mining tools as well, so you might want to add a check to avoid those.

Reply:
These are sorted by the category of knife skins on Hypixel.